x-pack/server/lib/check_license/check_license.js

```diff
@@ -4,29 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export function checkLicense(xpackLicenseInfo) {
-  const pluginName = 'Index Management';
-
+export function checkLicense(pluginId, validLicenseModes = [], xpackLicenseInfo) {
   // If, for some reason, we cannot get the license information
   // from Elasticsearch, assume worst case and disable
   if (!xpackLicenseInfo || !xpackLicenseInfo.isAvailable()) {
     return {
       isAvailable: false,
       showLinks: true,
       enableLinks: false,
-      message: `You cannot use ${pluginName} because license information is not available at this time.`
+      message: `You cannot use ${pluginId} because license information is not available at this time.`
```

Comment:
I think we want to keep this as `pluginName` (and update the consumer to provide the plugin name, not the ID), to keep as a human-readable message: "You cannot use Index Management..." vs "You cannot use index_management..."